packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -167,7 +177,14 @@ export const VListItem = genericComponent<VListItemSlots>()({
       if (isGroupActivator || !isClickable.value) return
 
       link.navigate?.(e)
-      props.value != null && select(!isSelected.value, e)
+
+      if (root.activatable) {
```

Comment:
I think this (and `root.selectable`) should use `.value`, because it's a Ref.

(I just started using Vuetify (again) today, and am using a selectable v-list. It's not activatable, but the code still calls `activate()`)